src/components/dialogs/stashed-elements/stashed-elements-dialog.tsx

```diff
@@ -74,9 +74,20 @@ const StashedElementsDialog = ({
         deleteElements(selectedElements, directoryToRestore.elementUuid)
             .then(onStashedElementChange)
             .catch((error) => {
-                snackError({
-                    messageTxt: error.message,
-                });
+                if (error.status === 403) {
+                    const errorMsg = intl.formatMessage(
+                        { id: 'DeleteElementFromStashError' },
+                        { multiselect: selectedElements.length > 1 }
```

Comment:
selectedElements could be null
selectedElements?.length

Reply:
no it can not: const [selectedElements, setSelectedElements] = useState<string[]>([]);